azurerm/internal/services/netapp/netapp_volume_resource.go

```diff
@@ -118,6 +118,17 @@ func resourceNetAppVolume() *schema.Resource {
 				},
 			},
 
+			"security_style": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"Unix",
+					"Ntfs",
```

Comment:
could we use the SDK build in constants here?

Reply:
We need to leave as is since the backend is changing the values to pascal case which differs from the sdk.